### DISCUSSION TITLE
Timings log

### DISCUSSION THREAD

**Discussion by rickinhissmart** [AntonyCorbett/OnlyT#526] [Ideas]
Is it possible to save a text log file at the end of meeting to show each item if time went over please ?

**Comment by AntonyCorbett** [comment]
Please let us know what you are trying to achieve and how the Timing Report fails to meet the need, Thanks, Antony

**Comment by AntonyCorbett** [comment]
@rickinhissmart Thanks. Does the standard timing report provide what you need?

See https://github.com/AntonyCorbett/OnlyT/wiki/Timing-Reports

**Reply by rickinhissmart** [reply]
Thank you I didn't know about the function I will try on Sunday

**Comment by rickinhissmart** [comment]
Thank you this feature works very well
